### ISSUE TITLE
Add the possibility to see the original duration for a step

### ISSUE THREAD

**Issue by paymand** [damianszczepanik/cucumber-reporting#265]
The duration for each step is in nanoseconds (at least in my json files that are produced by cucumber-js).
If I understand it correctly, they are rounded down to milliseconds which makes us lose precision.
For fast steps like the hooks, the duration is sometimes rounded down to 0 which is not very helpful in my opinion.

Personally, I think the precision information shouldn't be lost. I suggest that you keep the original values in your template so that it is possible to convert them using javascript on the client-side:

In other words, display the times in milliseconds by default, but provide the possibility to convert the values to microseconds or back to original values in nanoseconds if one desires.

**Comment by damianszczepanik** [comment]
I believe I can close this after pull #266
